projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -258,6 +258,18 @@ datajobs.aws.defaultSessionDurationSeconds=${DATAJOBS_AWS_DEFAULT_SESSION_DURATI
 
 # Hashicorp Vault Integration settings
 # When disabled/not configured the Secrets functionality won't work
+# In production, we would only use AppRole Authentication which require setup in vault:
+# https://developer.hashicorp.com/vault/docs/auth/approle
+# https://developer.hashicorp.com/vault/tutorials/auth-methods/approle
+# and should provide roleid and secretid to the service.
+#
+# For local development you can start a vault server with the following command:
+# vault server -dev -dev-root-token-id="root"
+# and configure only the uri and the token
 featureflag.vault.integration.enabled=false
-vdk.vault.uri=http://localhost:8200
+# If you get 404 errors related to vault operations, double-check the vault URI, it should usually end with "/v1/"
+vdk.vault.uri=http://localhost:8200/v1/
+vdk.vault.approle.roleid=feds259142-9b53-d163-1ae9-d6286d3dfe22
+vdk.vault.approle.secretid=ddsac36f9e9-1555-8c29-9a7f-283608d06fbd
```

Comment:
What are those 2 values ? If they won't work for anyone else better to be empty.

Reply:
fixed.